src/core/services/VulnerabilityFindings.ts

```diff
@@ -0,0 +1,9 @@
+import { BaseService, PaginatedRequestOptions, RequestHelper } from '../infrastructure';
+
+export class VulnerabilityFindings extends BaseService {
+  all(projectId: string | number, options?: PaginatedRequestOptions) {
```

Comment:
Perhaps add the types for the query options?

Reply:
Updated with what I think you were after.  I haven't done anything with Typescript before so this is all a bit new, let me know if that's not what you were thinking.

Having said that I'm a noob on TS I did wonder if making things like the `severity`, `confidence` and `report_type` into enums would make sense here?  I didn't see any enums anywhere in the project so wasn't sure if there was a reason to avoid them or not.

Reply:
in ts, enum is as simple as saying `'undefined' | 'ignore'` etc :)

enums do exist in typescript: https://www.typescriptlang.org/docs/handbook/enums.html but in this case I think possible values are faster.

Reply:
otherwise LGTM

Reply:
Nice addition! I would suggest making the properties camelCase for consistency. Other than that LGTM

Reply:
I’ll change to camelCase, that was a result of me being consistent with the gitlab API, my bad.

Reply:
we got the library that takes care of the casing 😉

Reply:
updated with camelCase